### PR TITLE
button component

### PR DESCRIPTION
**Pull request by devAmlan**
This is the button component. Please review my PR.

### CODE REVIEW

**Review by developer-anand — APPROVED** [review]
Good work man

File: library/component/button/button.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      position: fixed;
+      bottom:1rem;
+      right:1rem;
+    }
+    .primary_floatbtn{
+       background-color:var(--primary-bgcolor);   
+       color:var(--dark)
+    }
+   .icon_btn{
+      padding:0.5rem 2.1rem;
```
Comment (zoef-yt) [inline library/component/button/button.css]: use px here

File: library/component/button/button.css [hunk 2]
```diff
@@ -0,0 +1,97 @@
+.btn{
+    padding:0.5rem 2.1rem;
```
Comment (zoef-yt) [inline library/component/button/button.css]: use px here

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: library/component/button/button.css (hunk 1 above)
Comment (devAmlan) [reply library/component/button/button.css]: done,  thanks zoef

**Pull request merged**